### PR TITLE
Revise description of Pariksha in README.

### PR DESCRIPTION
**Pull request by htjjen**
Revise description by adding more information and changing the order of sentences.

Issue #3

### CODE REVIEW

**Comment by klbm9999** [comment]
Apart from that it presently has these features : 
1. Randomly generates questions set from the database
2. Evaluates responses immediately and outputs score, doesn't require human intervention.

To install  it, we could do the following (assuming the question-answer set exists in database) : 
1. Copy the repo into your webroot.
2. In scripts/config.php change the database details (is username, password and dbname)
3. Change the `number_of_questions` parameter in config file.
4. Open the URL http://your_webroot/quiz.php

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+## Getting Started
+### Prerequisites
+The question-answer set should exist in the database before installation.
+
+### Installing
+1. Copy the Pariksha repository into your webroot.
+2. In the file [scripts/config.php](https://github.com/klbm9999/Pariksha/blob/master/scripts/config.php):
+   * specify the details of the database, such as the username, password, and database name; and
+   * specify the `no_of_questions` parameter.
+3. Open the [URL](http://your_webroot/quiz.php).
```
Comment (klbm9999) [inline readme.md]: the url in last point is not a link, it is a type of link, some thing like this : 
- Open the url `http://your_webroot/quiz.php`

ie `your_webroot` is  the users link where he copied the files

**Comment by klbm9999** [comment]
The content up until now is fine, i would like you to frame some questions regarding the project, which we will answer and that would probably be a better approach to creating the readme.
I will also the 'under development' features soon

**Comment by htjjen** [comment]
Hi @klbm9999 , 

Thanks for the suggestion regarding the URL. I have made the change. 

With respect to the questions, that's a great idea! I would like to ask some questions at Issue #3 before I start writing these questions. In the meantime, is it possible for you to approve this PR #8 while I'm working on the questions? My assignment deadline is approaching and it's important for me to have my PRs approved and merged. Please let me know. Thank you! :)

**Comment by klbm9999** [comment]
Sure why not, its done.

**Comment by htjjen** [comment]
@klbm9999  Thank you very much!

**Pull request merged**